src/assets/inputFile/input.json

```
@@ -52,16 +52,21 @@
       "ybr": "Your Ballot Is Sealed",
       "bfp": "Ballot Checking Code:",
       "zyx098": "zyx098-wvu765-tsr432-1234",
-      "ydn": "To check your ballot, go to VerifyMyBallot.gov and enter the Ballot Checking Code displayed above. After you enter the code, tap the Code Entered button below.",
+      "first_para_text_bfp": "Submit your ballot now, or perform a ballot check using the instructions below.",
+      "ydn": "To check your ballot, go to VerifyMyBallot.gov and enter the Ballot Checking Code displayed above. ",
+      "second_para_text": "After you enter the code, tab the Code Entered button below.",
+      "tv4": "Trouble entering the code?",
+      "report": "Report a problem below.",
       "copy": "Code Entered ",
-      "tv3_fb": "For more information on testing, click the link below.",
-      "link": "www.linkto.site.org",
+      "tv3_fb": "For more information on ballot checking, visit",
+      "link": "electionwebsite.gov",
       "tv_fp": "Or, you can cast your ballot digitally now.",
-      "send": "Cast Ballot Now",
+      "send": "Submit Ballot Now",
       "shmd": "Show more details",
       "tv1_fp": " On a computer or other device,go to",
       "tv2_fb": "and enter  the  following code to review your ballot",
-      "mp": "My Progress"
+      "mp": "My Progress",
+      "rap": "Report  A Problem"
```

Should there only be a single space here?

```suggestion
      "rap": "Report A Problem"
```

---

I have removed double space from the string in input.json file